configs/ld/readme.md

```diff
@@ -0,0 +1,33 @@
+# Localization Distillation for Object Detection
+
+## Introduction
+
```

Comment:
Need a tag `[ALGORITHM]` here for model zoo statistics. See example in [faster_rcnn](https://github.com/open-mmlab/mmdetection/tree/master/configs/faster_rcnn).